development/README.md

```diff
@@ -282,4 +282,191 @@ $BUILD_SCRIPTS/qgis-dev-install.sh $BUILD_DIR
 
 ## Configure/build/install QGIS in Qt Creator.app
 
-TODO
+### Install QtCreator 4.0 or better
+
+* Grab the installer from https://www.qt.io/download-open-source/
+* Run the installer. You ONLY need the qt-creator package so disable the Qt5 and don’t install Qt libs (we will use the qt4 installed by brew)
+* Once the installer is run open qt-creator
+* Make a directory in your QGIS source tree called ``build``
```

Comment:
The directory further down says `build-relwithdebinfo` is that intended to be a different one?
Anyway, I think QtCreator will create a build directory without any manual intervention, no?